### DISCUSSION TITLE
my mobile alleys hacked

### DISCUSSION THREAD

**Discussion by Rcpsumon** [community/community#145670] [Mobile]
### Select Topic Area

Show & Tell

### Body

I study mobile and Internet access and I have to pc the data from my recovery system and it will help 😔 I have a oiliness and Internet  usage no idea so I samthin English tell and rating 😳 helps me stady plz sir

**Comment by Marco1981s** [comment]
> ### Select Topic Area
> 
> Show & Tell
> 
> ### Body
> 
> I study mobile and Internet access and I have to pc the data from my recovery system and it will help 😔 I have a oiliness and Internet  usage no idea so I samthin English tell and rating 😳 helps me stady plz sir
